.gitignore

```diff
@@ -71,4 +71,6 @@ yarn-error.log
 # Yarn Integrity file
 .yarn-integrity
 
-static/
\ No newline at end of file
+static/
+
+.idea
```

Comment:
just curious what this is for, local noodling?

Reply:
yes. .idea is generated by any of the jetbrains editors. (I use Webstorm instead of vs code)